This is minor but I got some compile warnings on a deprecated header

```
In file included from /root/industrial_moveit_ws/src/industrial_moveit/stomp_moveit/src/stomp_planner.cpp:29:0:
/opt/ros/melodic/include/class_loader/class_loader.h:36:2: warning: #warning Including header <class_loader/class_loader.h> is deprecated, include <class_loader/class_loader.hpp> instead. [-Wcpp]
 #warning Including header <class_loader/class_loader.h> is deprecated, \
  ^~~~~~~
In file included from /root/industrial_moveit_ws/src/industrial_moveit/stomp_moveit/src/stomp_planner_manager.cpp:26:0:
/opt/ros/melodic/include/class_loader/class_loader.h:36:2: warning: #warning Including header <class_loader/class_loader.h> is deprecated, include <class_loader/class_loader.hpp> instead. [-Wcpp]
 #warning Including header <class_loader/class_loader.h> is deprecated, \

.
.
.

                 from /root/industrial_moveit_ws/src/industrial_moveit/constrained_ik/src/moveit_interface/constrained_ik_planner_plugin.cpp:26:
/opt/ros/melodic/include/class_loader/class_loader.h:36:2: warning: #warning Including header <class_loader/class_loader.h> is deprecated, include <class_loader/class_loader.hpp> instead. [-Wcpp]
 #warning Including header <class_loader/class_loader.h> is deprecated, \

```

--- a/constrained_ik/src/basic_kin.cpp
+++ b/constrained_ik/src/basic_kin.cpp
@@ -185,7 +185,7 @@ bool BasicKin::init(const moveit::core::JointModelGroup* group)
   }
 
   const robot_model::RobotModel& r  = group->getParentModel();
-  const boost::shared_ptr<const urdf::ModelInterface> urdf = group->getParentModel().getURDF();
+  const urdf::ModelInterfaceSharedPtr urdf = group->getParentModel().getURDF();
   base_name_ = group->getLinkModels().front()->getParentLinkModel()->getName();
   tip_name_ = group->getLinkModels().back()->getName();
 

--- a/constrained_ik/src/moveit_interface/constrained_ik_plugin.cpp
+++ b/constrained_ik/src/moveit_interface/constrained_ik_plugin.cpp
@@ -62,8 +62,8 @@ bool ConstrainedIKPlugin::initialize(const std::string& robot_description,
 
   // init robot model
   rdf_loader::RDFLoader rdf_loader(robot_description_);
-  const boost::shared_ptr<srdf::Model> &srdf = rdf_loader.getSRDF();
-  const boost::shared_ptr<urdf::ModelInterface>& urdf_model = rdf_loader.getURDF();
+  const srdf::ModelSharedPtr &srdf = rdf_loader.getSRDF();
+  const urdf::ModelInterfaceSharedPtr& urdf_model = rdf_loader.getURDF();
 
   if (!urdf_model || !srdf)
   {

--- a/constrained_ik/src/moveit_interface/joint_interpolation_planner.cpp
+++ b/constrained_ik/src/moveit_interface/joint_interpolation_planner.cpp
@@ -63,7 +63,11 @@ namespace constrained_ik
     const robot_model::JointModelGroup *group_model = rob_model->getJointModelGroup(request_.group_name);
     std::vector<std::string> joint_names = group_model->getActiveJointModelNames();
     std::vector<std::string> link_names = group_model->getLinkModelNames();
-    Eigen::Affine3d goal_pose;
+    #if ROS_VERSION_MINIMUM(1, 14, 0) //melodic
+      Eigen::Isometry3d goal_pose;
+    #else
+      Eigen::Affine3d goal_pose;
+    #endif
     std::vector<double> pos(1);
     robot_trajectory::RobotTrajectoryPtr traj(new robot_trajectory::RobotTrajectory(rob_model, request_.group_name));
 

--- a/industrial_collision_detection/CMakeLists.txt
+++ b/industrial_collision_detection/CMakeLists.txt
@@ -17,7 +17,6 @@ find_package(catkin REQUIRED COMPONENTS
   cmake_modules
   pluginlib
   moveit_core
-  pcl_ros
 )
 
 ###################################
@@ -32,8 +31,8 @@ catkin_package(
  INCLUDE_DIRS include ${catkin_INCLUDE_DIRS}
  LIBRARIES
    ${PROJECT_NAME}
- CATKIN_DEPENDS roscpp moveit_core cmake_modules pluginlib pcl_ros
- DEPENDS Boost EIGEN3
+ CATKIN_DEPENDS roscpp moveit_core cmake_modules pluginlib
+ DEPENDS Boost EIGEN3 LIBFCL
 )
 
 ## Specify additional locations of header files

--- a/industrial_collision_detection/package.xml
+++ b/industrial_collision_detection/package.xml
@@ -15,13 +15,13 @@
   <build_depend>moveit_core</build_depend>
   <build_depend>cmake_modules</build_depend>
   <build_depend>pluginlib</build_depend>
-  <build_depend>pcl_ros</build_depend>
+  <build_depend>libfcl-dev</build_depend>
 
   <run_depend>roscpp</run_depend>
   <run_depend>moveit_core</run_depend>
   <run_depend>cmake_modules</run_depend>
   <run_depend>pluginlib</run_depend>
-  <run_depend>pcl_ros</run_depend>
+  <run_depend>libfcl-dev</run_depend>
   
 
   <!-- The export tag contains other, unspecified, tags -->

